manager/manager.go

```diff
@@ -689,6 +689,29 @@ func (m *manager) GetProcessList(containerName string, options v2.RequestOptions
 	return ps, nil
 }
 
+func (m *manager) registerCollectors(collectorConfigs map[string]string, cont *containerData, cm collector.CollectorManager) error {
+	for k, v := range collectorConfigs {
+		configFile, err := cont.ReadFile(v, m.inHostNamespace)
+		if err != nil {
+			glog.Infof("failed to read config file %q for config %q, container %q: %v", v, k, cont.info.Name, err)
```

Comment:
Remove the glog's and merge it into the error. We already log the error in the calling function